osu.Game/Rulesets/Mods/ModTimeRamp.cs

```diff
@@ -26,6 +26,9 @@ public abstract class ModTimeRamp : Mod, IUpdatableByPlayfield, IApplicableToBea
         [SettingSource("Final rate", "The final speed to ramp to")]
         public abstract BindableNumber<double> FinalRate { get; }
 
+        [SettingSource("Adjust Pitch", "Should pitch be adjusted with speed")]
```

Comment:
The casing of the setting isn't consistent with the other names (although setting name casing is all over the place right now and probably needs to be consolidated at some point).